### ISSUE TITLE
[Feature Request] Please provide a way to not include XMP metadata

### ISSUE THREAD

**Issue by rbrito** [pikepdf/pikepdf#93]
Dear @jbarlow83,

I have a script (that I am willing to send you as examples of uses of pikepdf) that removes many kinds of metadata (which, in particular also help when optimizing the resulting PDF files with [pdfsizeopt][0]).

Unfortunately, even if I delete all kinds of `/Metadata` (I can provide the code) and, then, save the file, the resulting file still has a "minimal" XMP that reduces the possibility of making the files one step closer to being anonymized:

```
$ pdfinfo -meta foo.clean.pdf
<?xpacket begin="﻿" id="W5M0MpCehiHzreSzNTczkc9d"?>
<x:xmpmeta xmlns:x="adobe:ns:meta/" x:xmptk="pikepdf">
 <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
 </rdf:RDF>
</x:xmpmeta>

<?xpacket end="w"?>

```

Would it be possible to avoid including this empty XMP metadata? Perhaps by a keyword to `pikepdf._qpdf.Pdf.save` like `add_xmp` so that the user can call it like:

```python
my_pdf.save('foo.pdf', add_xmp=False)
```
Thank you very much for this wonderful library,

Rogério Brito.

[0]: https://github.com/pts/pdfsizeopt

**Comment by jbarlow83** [comment]
Are you saving with .save(fix_metadata_version=False) ?

**Comment by rbrito** [comment]
> Are you saving with .save(fix_metadata_version=False) ?

Thanks for the suggestion. The documentation wasn't clear if such metadata would be created if it were empty and I didn't set `fix_metadata_version=False`.

Setting `fix_metadata_version=False` deals with the XMP metadata. Thanks. OTOH, I see that the files have an infodict even if it is empty (and even if I call `del` on it before saving)...

**Comment by jbarlow83** [comment]
I think it is cleaner to not create an empty XMP object so I will address that.

You can del use `del pdf.trailer.Info` to delete the document info trailer. ``pdf.docinfo`` is a helper-wrapper that you cannot delete.

**Comment by jbarlow83** [comment]
Fixed for v1.10.1
